src/App.js

```diff
@@ -26,12 +30,53 @@ class App extends React.Component {
         wordDisplay.push("_");
       }
     }
+
     return wordDisplay.toString();
   };
 
+  checkIfUserWon = () => {
+    for (let letter of this.state.currWord) {
+      if (this.state.guessedLetters.includes(letter)) continue;
+      else {
+        return false;
+      }
+    }
+    return true;
+  };
+  checkIfUserLost = () => {
+    const hasUserWon = this.checkIfUserWon();
+    if (!hasUserWon && this.state.numOfGuessesLeft === 0) return true;
```

Comment:
```suggestion
    if (!this.checkIfUserWon() && !this.state.numOfGuessesLeft) return true;
```

no need to use a variable if we don't reuse the value and it is clear what the function is doing